test/integration/transaction.test.js

```diff
@@ -73,6 +73,30 @@ describe(Support.getTestDialectTeaser('Transaction'), function() {
         expect(t.finished).to.be.equal('rollback');
       });
     });
+
+    if (dialect === 'postgres' || dialect === 'mssql') {
```

Comment:
Mysql returns the values and does not throw. But that does not really matter as my fix works for every dialect if something like that happens in that dialect.